docs/reference/troubleshooting/disk/fix-data-node-out-of-disk.asciidoc

```diff
@@ -0,0 +1,23 @@
+[[fix-data-node-out-of-disk]]
+== Fix data nodes out of disk
+
+{es} is using data nodes to distribute your data inside the cluster. If one or more of these nodes are running
+out of space, {es} takes action to redistribute your data within the nodes so all nodes have enough available
+disk space. If {es} cannot facilitate enough available space in a node, then you will need to intervene in one
+of two ways:
+
+. <<increase-capacity-data-node, Increase the disk capacity of your cluster>>
+. <<decrease-disk-usage-data-node, Reduce the disk usage by decreasing your data volume>>
+
+[[increase-capacity-data-node]]
+=== Increase the disk capacity of data nodes
+include::{es-repo-dir}/tab-widgets/troubleshooting/disk/increase-data-node-capacity-widget.asciidoc[]
+
+[[decrease-disk-usage-data-node]]
+=== Decrease the disk usage of data nodes
+In order to decrease the disk usage in your cluster without losing any data is to reduce the replicas of indices.
```

Comment:
It seems part of this sentence is missing. Did you mean "In order to decrease the disk usage in your cluster without losing any data, the best way is to reduce the replicas of indices."?

Reply:
Good point, I will rephrase.